packages/esignet-mock/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { cleanEnv, port, str, url } from "envalid";
+
+export const env = cleanEnv(process.env, {
+  PORT: port({ default: 20260 }),
+  HOST: str({ default: "0.0.0.0", devDefault: "localhost" }),
+  CLIENT_URL: url({ devDefault: "http://localhost:3000" }),
+  });
```

Comment:
```suggestion
});
```
hmmm have I might have forgot to install Prettier to this repo. I can action on that